core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -333,9 +337,12 @@ private static String formatFragment(
             double sdAmongTasks = Math.sqrt(squaredDifferences / stageInfo.get().getTasks().size());
 
             builder.append(indentString(1))
-                    .append(format("CPU: %s, Scheduled: %s, Input: %s (%s); per task: avg.: %s std.dev.: %s, Output: %s (%s)\n",
+                    .append(format("CPU: %s, Scheduled: %s, Blocked %s (Input: %s, Output: %s), Input: %s (%s); per task: avg.: %s std.dev.: %s, Output: %s (%s)\n",
```

Comment:
I am a bit concerned if it is readable. We have `Input:` and `Output:` in the string twice. But maybe it is fine.

Reply:
Yeah, I also thought about that :-) Though ran out of ideas how to make it more readable. Basically i was considering a couple of options:

- Moving the `Blocked` section to the end of the line. But then it feels like `CPU / Scheduled / Blocked` should be next to each other.
- Instead of spelling `Input: ... Output: ...` I was also thinking of simplifying it to `(I: ..., O: ...)` (since it's rather a "niche" information and one looking is assumed to already understand what they are looking at). But then it felt like it is still better to be more explicit so it is less confusing for somebody who doesn't know completely what it is about. 

Happy to hear your though

Reply:
Maybe `Blocked total/input/output`? But it is even less explicit than `I:.., O:...`.
I think we can keep it. Having this info in single line implies that it will be not super trivial to parse. And I do not think we want to split it to multiple lines (surely not in this PR).